cql3/statements/create_aggregate_statement.cc

```diff
@@ -42,6 +42,13 @@ shared_ptr<functions::function> create_aggregate_statement::create(query_process
         throw exceptions::invalid_request_exception(format("State function '{}' doesn't return state", _sfunc));
     }
 
+    ::shared_ptr<cql3::functions::scalar_function> reduce_func = nullptr;
+    if(_rfunc) {
+        reduce_func = dynamic_pointer_cast<functions::scalar_function>(functions::functions::find(functions::function_name{_name.keyspace, _rfunc.value()}, {state_type, state_type}));
+        if (!reduce_func) {
+            throw exceptions::invalid_request_exception(format("Reduce function not found: {}", _rfunc));
```

Comment:
The problem could be that it wasn't found (for specific parameter types), or that it was not a scalar function. The error can be more helpful.

Reply:
Added state type in error message